pkg/crc/validation/validation.go

```diff
@@ -69,7 +69,7 @@ func ValidateBundle(bundle string) error {
 	releaseBundleVersion := version.GetBundleVersion()
 	userProvidedBundleVersion := filepath.Base(bundle)
 	if !strings.Contains(userProvidedBundleVersion, fmt.Sprintf("%s.crcbundle", releaseBundleVersion)) {
-		if !constants.BundleEmbedded() {
+		if present, _ := constants.BundlePresent(); !present {
 			logging.Warnf("Using unsupported bundle %s", userProvidedBundleVersion)
```

Comment:
We were using `!constants.BundleEmbedded()` as a way to differentiate between releases and builds which are not release binaries. I don't think the same logic still works when the bundle is no longer embedded.

Reply:
@cfergeau Yes we can remove this logic in that case we are not going to return the error message but only the `Warnf` message.